test/Driver/linker-order.ll

```diff
@@ -0,0 +1,24 @@
+; RUN: llc -mtriple armv7--linux-gnueabihf -filetype obj -o - %s | swift-autolink-extract -o - - | FileCheck %s
+; REQUIRES: autolink-extract
```

Comment:
Oh, I didn't even see that.  I like that idea very much.  Ill upload a change with that moved.